### DISCUSSION TITLE
Alarms randomly not raised, even if the logs say the opposite

### DISCUSSION THREAD

**Discussion by dagord** [NightscoutFoundation/xDrip#2190] [Q&A]
I'm using xDrip+ for the last three weeks and I'm happy with it.
By the way, I'm experiencing a random issue: when the blood sugar level exceeds the threshold (either low or higher), the alarm are not raised.
I get a notification on the phone, I get a notification on my MiBand 4 too, the alarm is recorded into the logs but no sound is actually played.
It just "unstucks" spontaneously after a random time, otherwise I need to reboot the phone.

This happened four/five times; please see the attached screenshot as examples: the first episode occoured during night, so I could be in deep spleeping, but on the second time I was wide awake so I could double check.

The phone itself is a Redmi Note 7 (MIUI Global 12.5.1, Android 10 QKQ1.190910.002); no silent or DnD modes are enabled and, by the way, other apps work normally.

XDrip+ is updated to the last stable version (e557cba-2022.05.19) but the issue occured with the previous one too.

I'm using a Dexcom g6 sensor, if this makes difference.

![1656322229927](https://user-images.githubusercontent.com/21258057/175918791-1ff05843-0ad3-4830-9290-f7a9d7c0af10.jpg)
![1656322311619](https://user-images.githubusercontent.com/21258057/175918808-7ec10977-6243-49f2-a832-f9b0a5b226b2.jpg)

**Comment by Navid200** [comment]
Can you also show the alert setting page?

**Reply by dagord** [reply]
As an example, this is the "high sugar level" alarm
![Screenshot_2022-06-27-16-15-13-785_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/21258057/175962976-3acbbc77-e607-4d09-a2df-522c26d0e811.jpg)

**Reply by Navid200** [reply]
Is there a reason override silent mode is disabled?

**Reply by dagord** [reply]
Yes, I don't want to be bothered in certain circumstances (e.g. when I'm at a meeting)

**Reply by Navid200** [reply]
Can you enable Force speaker and see if it makes a difference?

**Reply by dagord** [reply]
I already enabled this (after publishing the post) for further testing and it's working so far, but it's difficult to say if it solved the issue as it's randomic.

**Reply by dagord** [reply]
Just an update: with the "force speaker" option enabled the issue has no longer occurred. Fingers crossed!
